src/Pact/Native.hs

```diff
@@ -92,7 +92,7 @@ listA = mkTyVar "a" [TyList (mkTyVar "l" []),TyPrim TyString,TySchema TyObject (
 enforceDef :: NativeDef
 enforceDef = defNative "enforce" enforce
   (funType tTyBool [("test",tTyBool),("msg",tTyString)])
-  "Fail transaction with MSG if pure function TEST fails, or returns true. \
+  "Fail transaction with MSG if value TEST is false. Otherwise, returns true. \
```

Comment:
Suggest "pure expression TEST evaluates to `false`" as "value" doesn't give correct intuition. Also, "pure" is significant here, I can't remember if we have a section describing "pure" functions above but keyset predicates and `enforce`, `enforce-one` require pure functions, ie without database access. We should document that somewhere

Reply:
As far as the function arguments go, the rendered docs are just asking for a `bool`. They don't seem to care how it came about. 

I agree though that `pure expression` is much better wording.